src/main/java/racingcar/view/input/ConsoleInputView.java

```diff
@@ -0,0 +1,11 @@
+package racingcar.view.input;
+
+import camp.nextstep.edu.missionutils.Console;
+
+public class ConsoleInputView extends InputView {
+    @Override
+    public String read() {
+        final String readFromConsole = Console.readLine();
+        return readFromConsole;
```

Comment:
```suggestion
        return Console.readLine()
```

이렇게 선언과 동시에 리턴하는 부분은 바로 리턴하는 방식으로 축약해도 깔끔할 것 같아요!

Reply:
@h-beeen 이 부분도 고민되더라고요~ 혹자는 브레이크포인트 디버깅을 위해 `return` 문에 쓰지 말라고 하고, 누군가는 코드의 깔끔함을 위해 `return` 문으로 바로 쓰는 걸 선호한다고 합니다. 저는 디버깅이 필요할 수도 있는 지점은 변수를 만들고, 간단한 boolean 반환은 바로 `return` 문에 작성하는데요. @h-beeen 님은 어떻게 생각하시나요~?

어떤 커뮤니티에서 농담으로 변수로 따로 안 만든 사람 찾아가고 싶다는 이야기를 들은 적도 있었어요. 🤣🤣
그 게시글에서도 의견이 분분해서 다른 분들의 의견도 궁금하네요.